CustomDynamicFont/Sources/Presentation/Modules/HomeViewController.swift

```diff
@@ -18,5 +18,7 @@ final class HomeViewController: UIViewController {
             helloLabel.centerYAnchor.constraint(equalTo: view.centerYAnchor),
             helloLabel.centerXAnchor.constraint(equalTo: view.centerXAnchor)
         ])
+
+        title = "Home"
```

Comment:
I agree. This will be done when we actually have a working vc. The home vc is a placeholder and will be delete later. https://github.com/nimblehq/ios-custom-font-scaling-demo/pull/29/files